string commandID
float64[] data
